pkg/sdk/network_policies_gen.go

```diff
@@ -133,7 +134,7 @@ type describeNetworkPolicyDBRow struct {
 	Value string `db:"value"`
 }
 
-type NetworkPolicyDescription struct {
+type NetworkPolicyProperty struct {
```

Comment:
they do not have the rest of the attributes that security integrations have?

Reply:
It's only key-value here